fetch.bs

```diff
@@ -5248,48 +5251,61 @@ these steps:
    <li><p>Let <var>serviceWorkerStartTime</var> be the <a for=/>coarsened shared current time</a>
    given <var>fetchParams</var>'s <a for="fetch params">cross-origin isolated capability</a>.
 
-   <li><p>Set <var>response</var> to the result of invoking <a for=/>handle fetch</a> for
+   <li><p>Let <var>fetchResponse</var> be the result of invoking <a for=/>handle fetch</a> for
    <var>requestForServiceWorker</var>, with <var>fetchParams</var>'s
    <a for="fetch params">controller</a> and <var>fetchParams</var>'s
    <a for="fetch params">cross-origin isolated capability</a>. [[!HTML]] [[!SW]]
 
    <li>
-    <p>If <var>response</var> is non-null, then:
 
-    <ol>
-     <li><p>Set <var>fetchParams</var>'s <a for="fetch params">timing info</a>'s
-     <a for="fetch timing info">final service worker start time</a> to
-     <var>serviceWorkerStartTime</var>.
+    <p>If <var>fetchResponse</var> is non-null, then:
+     <ol>
+     <li><p>If<var>fetchResponse</var> is <a for=/>response</a>, then:
+      <ol>
+       <li><p>Set <var>response</var> to <var>fetchResponse</var>.
+       <li><p>Set <var>fetchParams</var>'s <a for="fetch params">timing info</a>'s
+       <a for="fetch timing info">final service worker start time</a> to
+       <var>serviceWorkerStartTime</var>.
+       <li><p>Set <var>fetchParams</var>'s <a for="fetch params">timing info</a>'s
+       <a for="fetch timing info">service worker timing info</a> to
+       <var>response</var>'s <a for="response">service worker timing info</a>.
```

Comment:
"2.2.6. Responses" should be updated to have associated `service worker timing info`?

Reply:
Thank you.  I overlooked that.  Then the sentence sounds reasonable.

Reply:
Ah, there seems not be the way to mark this resolved in my side.